e2e/cypress/support/api_commands.js

```diff
@@ -56,6 +56,54 @@ Cypress.Commands.add('apiEmailTest', () => {
 // https://api.mattermost.com/#tag/posts
 // *****************************************************************************
 
+/**
+* Creates a post directly via API
+* This API assume that the user is logged in and has cookie to access
+* @param {String} channelId - Where to post
+* @param {String} message - What to post
+* @param {String} rootId - Parent post ID. Set to "" to avoid nesting
+* @param {Object} props - Post props
+* @param {String} token - Optional token to use for auth. If not provided - posts as current user
+*/
+Cypress.Commands.add('apiCreatePost', (channelId, message, rootId, props, token = '', failOnStatusCode = true) => {
+    const headers = {'X-Requested-With': 'XMLHttpRequest'};
+    if (token !== '') {
+        headers.Authorization = `Bearer ${token}`;
+    }
+    return cy.request({
+        headers,
+        failOnStatusCode,
+        url: '/api/v4/posts',
+        method: 'POST',
+        body: {
+            channel_id: channelId,
+            root_id: rootId,
+            message,
+            props,
+        },
+    });
+});
+
+/**
+ * Creates a token directly via API
+ * This API assume that the user is logged in as admin
+ * @param {String} userId - user for whom to create the token
+ */
+Cypress.Commands.add('apiCreateToken', (userId) => {
```

Comment:
We're organizing API commands into `/support/api` folder and match file structure with `api.mattermost.com`. With that, could you please move this command to `/support/api/user.js` and add corresponding type at `/support/api/user.d.ts`? TIA